src/js/panels.js

```diff
@@ -29,7 +29,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         },
         step: 0.1,
         selectors: {
-            instructions: ".gpiic-fd-instructions",
+            rangeInstructions: ".gpiic-fd-instructions",
```

Comment:
msgBundle currently combines all json strings in one block. This causes the issue that, strings with the same message key overwrite each other. So each instruction key for different panels needs to be more specific.